src/cluster_legacy.c

```diff
@@ -7029,9 +7029,29 @@ int clusterCommandSpecial(client *c) {
         clusterDelNode(n);
         clusterDoBeforeSleep(CLUSTER_TODO_UPDATE_STATE | CLUSTER_TODO_SAVE_CONFIG);
         addReply(c, shared.ok);
-    } else if (!strcasecmp(c->argv[1]->ptr, "replicate") && c->argc == 3) {
-        /* CLUSTER REPLICATE <NODE ID> */
+    } else if (!strcasecmp(c->argv[1]->ptr, "replicate") && (c->argc == 3 || c->argc == 4)) {
+        /* CLUSTER REPLICATE (<NODE ID> | NO ONE)*/
         /* Lookup the specified node in our table. */
```

Comment:
please also move this comment to below, near the clusterLookupNode one.

Reply:
updated.